### PR TITLE
Support for multiple topics and types in Send Pose

### PR DESCRIPTION
**Pull request by Eman7C7**
Now Send Pose mode can support multiple topics of type Pose, PoseStamped and PoseWithCovarianceStamped.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -116,7 +116,9 @@ pose_stamped_topics:            # geometry_msgs::PoseStamped topics.
       g: 0
       b: 0
     length: 0.2                 # Length of the axes.
-send_pose_topic: initialpose    # Topic on which to publish poses in Send Pose mode.
+send_pose_topics:               # Topics on which to publish poses in Send Pose mode.
+  - topic: pose                 # The topic name.
+    msg_type: PoseStamped       # The topic's type. Supported are Pose, PoseStamped and PoseWithCovarianceStamped.
```
Comment (carzum) [inline README.md:121]: unacceptable trailing white space in this line :scream:

File: src/config.rs [hunk 2]
```diff
@@ -190,6 +199,8 @@ impl Default for TermvizConfig {
                 (input::ZOOM_OUT.to_string(), "-".to_string()),
                 (input::INCREMENT_STEP.to_string(), "k".to_string()),
                 (input::DECREMENT_STEP.to_string(), "j".to_string()),
+                (input::NEXT.to_string(), "n".to_string()),
```
Comment (carzum) [inline src/config.rs:202]: This is not consistent with the image mode. Maybe also use next and previous in that mode additionally?

File: src/config.rs [hunk 3]
```diff
@@ -190,6 +199,8 @@ impl Default for TermvizConfig {
...
+                (input::PREVIOUS.to_string(), "b".to_string()),
```
Comment (carzum) [inline src/config.rs:203]: previous should be N to be compliant with superior vim keymappings

**Review by carzum — CHANGES_REQUESTED** [review]
Works fine when testing, really good addition to send nav goals :+1:

File: src/config.rs (hunk 2 above)
Comment (Eman7C7) [reply src/config.rs:202]: I thought about it but I was not sure if it was in scope of the PR. I'll do it

File: src/config.rs (hunk 3 above)
Comment (Eman7C7) [reply src/config.rs:203]: do you want m for next and n for previous?

Comment (carzum) [reply src/config.rs]: on vim n is next match, N is previous match for searches

**Review by carzum — APPROVED** [review]

**Review by MaxiMaerz — APPROVED** [review]

**Pull request merged**